examples/build.gradle

```diff
@@ -30,6 +32,8 @@ group = "io.opencensus"
 version = "0.13.0-SNAPSHOT" // CURRENT_OPENCENSUS_VERSION
 
 def opencensusVersion = "0.12.2" // LATEST_OPENCENSUS_RELEASE_VERSION
+def grpcVersion = "1.9.0" // CURRENT_GRPC_VERSION
```

Comment:
We've upgraded grpc to 1.10.0.

Reply:
Actually we have to use gRPC v1.9.0 for this example now, because it's using the latest released version (v0.12.2), which depends on gRPC 1.9.0.

I have tested with gRPC v1.10.0 before - they have made some breaking changes in CensusModule.java, and running the example with v1.10.0 won't work.